pkg/apis/workflow/v1alpha1/workflow_types_test.go

```diff
@@ -255,53 +255,60 @@ func TestArtifact_ValidatePath(t *testing.T) {
 		a3 := Artifact{Name: "a3", Path: "../.."}
 		err = a3.CleanPath()
 		require.NoError(t, err)
-		assert.Equal(t, "../..", a3.Path)
+		assert.Equal(t, filepath.Join("..", ".."), a3.Path)
 
 		a4 := Artifact{Name: "a4", Path: "../etc/passwd"}
 		err = a4.CleanPath()
 		require.NoError(t, err)
-		assert.Equal(t, "../etc/passwd", a4.Path)
+		assert.Equal(t, filepath.Join("..", "etc", "passwd"), a4.Path)
 	})
 
+	ensureRootPathSeparator := func(p string) string {
```

Comment:
This probably makes more sense to convert to a plain named function as opposed to a variable assigned to an anonymous function since it doesn't use any scope

Reply:
I don't mind to adjust it, however a Go anon function is not really the same as UI anon functions I'd say. I simply opted for an anon function because it's only relevant to this test and there's no need to pollute the package namespace with that function, otherwise. 

but as said, I don't mind, happy to adjust it.

Reply:
adjusted.